D3D12HelloWorld/Games/StepTimer.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace wired.Games {
+    /// <summary>
+    /// Helper class for animation and simulation timing.
+    /// </summary>
+    internal class StepTimer {
```

Comment:
Straight copy of the class created for D3D12Bundles to port the QPC unit timing data, just updated the namespace.  Seems a useful type outside that sample.